main.js

```diff
@@ -0,0 +1,138 @@
+function validate(input) {
+    if (input === '' || input === ' ') {
+        alert ('A field should has a value. Try one more time!');
+    }
+}
+
+function validateIfNum(input) {
+    if (!Number.isInteger(input)) {
+    return false;
+    }
+}
+
+
+//1.
+const userYear = document.querySelector('.data-1');
+const btnFindoutCentury = document.querySelector('.btn-1');
+const resultCentury = document.querySelector('.result-1');
+
+function determineCentury(inputValue) {
+
+    let century = Math.floor(inputValue / 100) + 1;
+
+    switch(century % 10) {
+
+        case 1:
+            century = century + 'st';
+            break;
+        case 2:
+            century = century + 'nd';
+            break;
+        case 3:
+            century = century + 'rd';
+            break;
+        default:
+            century = century + 'th';
+    }
+    return century;
+}
+
+btnFindoutCentury.addEventListener('click', () => {
+    resultCentury.innerHTML = `Result: ${userYear.value} is a ${determineCentury(userYear.value)} century.`
+})
+
+//2.
+const usersRandomLetters = document.querySelector('.data-2');
+const btnTask2 = document.querySelector('.btn-2');
+const result2 = document.querySelector('.result-2');
+
+function conversionToNum(value) {
```

Comment:
All function should have names as in task
```suggestion
function vowelOne(value) {
```